indico/legacy/webinterface/common/tools.py

```diff
@@ -233,7 +233,7 @@ def escape_html(text, escape_quotes=False):
                           " => &quot;
                           ' => &#34;
     """
-    if type(text) != str:
+    if not isinstance(text, str):
```

Comment:
This is legacy code, not sure if it can get str subclasses that should not be converted. Safer to not touch this particular one. It's part of the very legacy HTML sanitizer that will soon go away anyway.